app/services/linkables_service.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require "gds_api/publishing_api_v2"
+
+class LinkablesService
```

Comment:
It might be good to take a look at content tagger and publisher for prior art:

https://github.com/alphagov/content-tagger/blob/master/app/models/linkables.rb
https://github.com/alphagov/publisher/blob/master/lib/tagging/linkables.rb

They do some additional things:

- There are some document type specific transformations (draft/no-draft filtering
- Use `internal_name` instead of the `title` (`internal_name` is meant for disambiguation in select boxes like this) 
- Showing "(draft)" in the title

Reply:
Good shout 👍